### DISCUSSION TITLE
Como configurar a exibição do Mapa?

### DISCUSSION THREAD

**Discussion by flyingml** [sepinf-inc/IPED#1972] [Q&A] [answered]
Quando clico na aba "Mapa" não aparece o mapa, apenas os placeholders (vide imagem). Como faço para configurar a exibição do mapa?
![image](https://github.com/sepinf-inc/IPED/assets/1653433/590f0360-5047-4ef9-a429-a9401429abf8)

**Comment by lfcnassif** [comment, marked as answer]
What version are you using? On 4.1.4 release notes there is a public announced fix for an issue like yours:
https://github.com/sepinf-inc/IPED/blob/4.1.4/ReleaseNotes.txt#L11

**Reply by flyingml** [reply]
I was using the 1.4.2 version. I updated to the latest version (1.4.5), now I can see the map, but not the placeholders.
![image](https://github.com/sepinf-inc/IPED/assets/1653433/705d800f-df82-4691-bbcc-34922a7d3c6e)

**Reply by flyingml** [reply]
I reopened the IPED and now the map is working fine. Thank you.
